xds/src/main/java/com/linecorp/centraldogma/xds/internal/XdsResourceWatchingService.java

```diff
@@ -85,6 +86,9 @@ protected void init() {
             logger.info("Creating xDS resources from {} at revision: {}", groupName, normalizedRevision);
             final Map<String, Entry<?>> entries = repository.find(normalizedRevision, pathPattern()).join();
             for (Entry<?> entry : entries.values()) {
+                if (entry.type() == EntryType.DIRECTORY || !entry.hasContent()) {
```

Comment:
Question) Considering only a JSON type is supported, would it make more sense to whitelist instead so that unexpected json deserialization failures could be avoided?

```suggestion
                if (entry.type() != EntryType.JSON || !entry.hasContent()) {
```

Reply:
That's a good suggestion. 👍